src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -638,6 +638,16 @@ public void initBinder(WebDataBinder webdataBinder) {
             .then(studyService.createEquipment(studyUuid, createLoadAttributes, ModificationType.LOAD_CREATION, nodeUuid)));
     }
 
+    @DeleteMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/{modificationUuid}")
+    @Operation(summary = "delete a network modification")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The network modification has been deleted")})
+    public ResponseEntity<Mono<Void>> deleteModification(@PathVariable("studyUuid") UUID studyUuid,
+                                                 @PathVariable("nodeUuid") UUID nodeUuid,
```

Comment:
`studyUuid` seems not used here ... why ?

Reply:
i do not need it

Reply:
Then why is it a PathVariable in this API ? Could we remove it ?

Reply:
to have the same path as the put (api rest...)